mrtt-ui/src/components/ProjectDetailsForm/ProjectDetailsForm.js

```diff
@@ -1,14 +1,15 @@
+import { useState } from 'react'
 import styles from './style.module.scss'
+import axios from 'axios'
 import { useForm, Controller } from 'react-hook-form'
 import { yupResolver } from '@hookform/resolvers/yup'
 import * as Yup from 'yup'
 import countries from '../../data/countries.json'
+import { projectDetailsMapping } from '../../data/questionMappingProjectDetails'
 import {
```

Comment:
Can you separate _external_ imports (react, react-hook-form, yup etc) from _internal_ imports specific to this project? It makes things much easier to find and understand 🙂

Reply:
yup! will keep external at the top and internal below